### DISCUSSION TITLE
Repeatable Jobs with Timezone

### DISCUSSION THREAD

**Discussion by manfe** [taskforcesh/bullmq#2363] [General]
Hey! I noticed that we can add a repeatable job informing the timezone using `tz` property.

But there is nothing related to this in the documentation, I will create a PR to add to the documentation if I have more information about it.

What are the supported values? Because the Typescript informs that requires a String for that.

Code example:

```js
      myQueue.add(
        "job-name-here",
        {}, // empty data
        {
          repeat: {
            pattern: '0 0 * * * *', // every hour
            tz: '<what should I put here?>'
          }
        }
      )
```

**Comment by manfe** [comment]
Probably the string format will be `America/Sao_Paulo` but I need a confirmation on that.

@manast when you have a chance. Thanks again.

**Reply by roggervalf** [reply]
hi @manfe, we are reusing extending this option from ParserOptions https://github.com/taskforcesh/bullmq/blob/master/src/interfaces/repeat-options.ts#L1 that comes from https://www.npmjs.com/package/cron-parser, I double check it a little bit and this package is using luxon as a dependency https://github.com/harrisiirak/cron-parser/blob/master/lib/date.js#L248 and this subdependency uses Intl https://github.com/moment/luxon/blob/master/src/zones/IANAZone.js#L7 to check tz formats, you can use this method to see all the available values https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/supportedValuesOf
